src/common/detectWMDE.c

```diff
@@ -275,6 +278,28 @@ static void getXFCE4(FFinstance* instance, FFWMDEResult* result)
     }
 }
 
+static void getLXQt(FFinstance* instance, FFWMDEResult* result)
+{
+    ffStrbufSetS(&result->deProcessName, "lxqt-session");
+    ffStrbufSetS(&result->dePrettyName, "LXQt");
+    ffParsePropFile("/usr/share/cmake/lxqt/lxqt-config.cmake", "set(LXQT_VERSION", &result->deVersion);
+
+    if(result->deVersion.length == 0 && instance->config.allowSlowOperations)
```

Comment:
Maybe we should try parsing other files before falling back to this. The other CMake file and the pkgconfig file look promising.